Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-image-gallery/index.jsx

```diff
@@ -58,11 +65,18 @@ const CloudinaryImageGallery = ({ size, cloudinaryImageGallery = {} }) => {
                         <>
                             <Box {...styles.heroImageGroup}>
                                 <AspectRatio {...styles.heroImage} ratio={1}>
-                                    <Img className={imageUrl?.isResponsive && 'cld-responsive'}
-                                        src={`${imageUrl.url.lastIndexOf('?') > -1 ? imageUrl.url.substring(0, imageUrl.url.lastIndexOf('?')) + cloudinary.CLD_TRACKING_PARAM : imageUrl.url + cloudinary.CLD_TRACKING_PARAM}`}
-                                        srcset={!imageUrl?.isResponsive && imageUrl?.srcset && updateTrackingParam(imageUrl?.srcset)}
-                                        sizes={!imageUrl?.isResponsive && imageUrl?.sizes}
-                                    />
+                                    {imageUrl?.isResponsive ? (
```

Comment:
please set the condition on line 64 to imageUrl (instead of imageUrls) and remove the questions marks further below, e.g. `imageUrl.§ instead of `imageUrl?.`